app/views/partials/foyer/patrimoine.html

```diff
@@ -140,11 +136,11 @@ <h2 class="panel-title">
         </div>
 
         <div class="form-group">
-          <label class="col-sm-4 control-label">Détenez-vous un <abbr title="Plan Épargne Logement">PEL</abbr>, un <abbr title="Plan d'épargne retraite populaire">PERP</abbr>, des titres, une assurance-vie, des parts d'une <abbr title="Société d'Investissement à CApital Variable">SICAV</abbr> ou d'un <abbr title="Fonds Commun de Placement">FCP</abbr> ?</label>
```

Comment:
Here, it was only `label`/`button` couple replaced, not the `form-group` altogether. This is inconsistent with the other parts.